src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerAdditionService.java

```diff
@@ -89,6 +89,10 @@ private IndividualBeneficialOwnerAddition getIndividualBeneficialOwnerAddition(
                         addressDtoToAddress(serviceAddress),
                         natureOfControls,
                         isOnSanctionsList);
+        
+        if (!bo.getTrustIds().isEmpty()) {
```

Comment:
We should protect against bo.getTrustIds() returning null. 
if (bo.getTrustIds() != null && !bo.getTrustIds().isEmpty()) {